### DISCUSSION TITLE
Add 'Standard' to input style types

### DISCUSSION THREAD

**Discussion by bzupancic** [IgniteUI/igniteui-angular#11169] [Ideas]
In addition to 'line', 'box', 'border', 'search', we would like to have a basic input style (label stacked on top of a bordered input) without having to use the Fluent theme or having to write a bunch of custom SCSS.

**Comment by simeonoff** [comment]
@bzupancic I am not sure we have plans for such design as it's not based on an official design language. Maybe @StefanIvanov can shed some more light on this.

**Comment by StefanIvanov** [comment]
Hello Brian, for the Material theme of Ignite UI for Angular we follow the guidance of the team supporting the [design language](https://material.io/components/text-fields) at Google and usually avoid introducing alternatives that are not in their system in the first place. Let us discuss internally and we will get back to you.
